auto_dev/commands/run.py

```diff
@@ -137,24 +149,42 @@ def check_tendermint(self, retries: int = 0) -> None:
         self.logger.info("Tendermint is running and healthy ✅")
         return None
 
-    def attempt_hard_reset(self, attempts: int = 0) -> None:
-        """Attempt to hard reset Tendermint."""
-        if attempts >= TENDERMINT_RESET_RETRIES:
-            self.logger.error(f"Failed to reset Tendermint after {TENDERMINT_RESET_RETRIES} attempts.")
-            sys.exit(1)
+    def start_tendermint(self, env_vars=None) -> None:
+        """Start Tendermint."""
+        self.logger.info("Starting Tendermint with docker-compose...")
 
-        self.logger.info("Tendermint is running, executing hard reset...")
+        env_vars = env_vars or {}
+        env_vars["FLASK_PORT"] = self.flask_port
```

Comment:
You don't actually need to change the flask port, just the mapping back to the host riggt?